CustomPlugin/StartupUtils.cs

```diff
@@ -0,0 +1,56 @@
+using System;
+using System.Threading.Tasks;
+using Decorators;
+using Environment;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+
+namespace CustomPlugin
+{
+    public class StartupUtils
+    {
+        public static readonly string DocsPath = "documentations";
```

Comment:
Why this string? I don't think its used anywhere

Reply:
It's used in the microservice template Startup to map swagger docs to this route

Reply:
ok. 

Please note that the standard route that we use to collect json swaggers should be `/documentation/json` so I guess this path should be without the final *s*, am I right? Can you check this out?

Reply:
Ok, I'm changing the path but I have some questions: does path `/documentation/json` never change? May it change, for example, due to updating from v1 to v2? Is Swagger UI necessary? If so, which path should be Swagger UI mapped to?